### DISCUSSION TITLE
What??????????????????????????????????

### DISCUSSION THREAD

**Discussion by Foxy-null** [Unlucky-Life/ankimon#22] [Show and tell]
![image](https://github.com/Unlucky-Life/ankimon/assets/72304646/61c26283-9bc9-4e18-9991-a59a5cc0ba19)

**Comment by Foxy-null** [comment]
BTW,
When I'm going to attack using Arceus, it returns this error.
![Screenshot 2024-03-15 163717](https://github.com/Unlucky-Life/ankimon/assets/72304646/a0d01f59-655e-44d3-a57d-5c93654abebb)

**Reply by Foxy-null** [reply]
The Arceus code (only for debugging):
![image](https://github.com/Unlucky-Life/ankimon/assets/72304646/fbe4a39e-eb46-48f6-97b1-ec8538444603)

**Reply by Unlucky-Life** [reply]
ok i believe this is due to the moves of arceus and propably some move having a special move stats and therefore not properly returning a damage value

**Comment by Unlucky-Life** [comment]
i just checked it - its the Move 'Teeter Dance' , which is somehow not able to befound.

**Comment by Unlucky-Life** [comment]
some attacks do work tho like charm or confusion
=> on the update on sunday there will be a remember move feature integrated => there you can remember a different move to help you remember a diffrent attack and replace this attack move or any move cause are currently sadly not supported
